unittests/osr_test.py

```diff
@@ -0,0 +1,153 @@
+# Copyright (c) 2017 CEF Python, see the Authors file.
+# All rights reserved. Licensed under BSD 3-clause license.
+# Project website: https://github.com/cztomczak/cefpython
+"""OSR testing of CEF Python."""
+
+import unittest
+import _test_runner
+from cefpython3 import cefpython as cef
+import platform
+import sys,time,os
+import base64
+
+g_datauri_data = """
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <script>
+        function print(msg) {
+            console.log(msg + " [JS]");
+        }
+        window.onload = function(){
+            document.addEventListener("mousedown", function( e ) {
+              print('mousedown:'+e.pageX+","+e.pageY)
+            }, false);
+            document.addEventListener("mouseup", function( e ) {
+              print('mouseup:'+e.pageX+","+e.pageY)
+            }, false);
+
+            document.addEventListener("mousemove", function( e ) {
+              print('mousemove:'+e.pageX+","+e.pageY)
+            }, false);
+
+            document.addEventListener("dragenter", function( e ) {
+              print('dragenter:'+e.pageX+","+e.pageY)
+            }, false);
+
+            document.addEventListener("dragover", function( e ) {
+              print('dragover:'+e.pageX+","+e.pageY)
+            }, false);
+            document.addEventListener("drop", function( e ) {
+              print('drop:'+e.pageX+","+e.pageY)
+            }, false);
+
+
+        }
+        print('window.onload init')
+    </script>
+    <style>
+        .box{
+            background-color: #F00;
+            height: 200px;
+        }
+    </style>
+</head>
+<body>
+
+<input id="selectFile" type="file">
+<div class="box"></div>
+</body>
+</html>
+
+"""
+
+g_datauri = "data:text/html;base64," + base64.b64encode(g_datauri_data.encode(
```

Comment:
Replace these code with a call to html_to_data_uri in both main_test.py and osr_test.py.